server/datastore/mysql/migrations/tables/20240813103800_SupportMultipleABMTokens_test.go

```diff
@@ -0,0 +1,86 @@
+package tables
+
+import (
+	"crypto/md5" //nolint:gosec
+	"testing"
+	"time"
+
+	"github.com/fleetdm/fleet/v4/server/ptr"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/require"
+)
+
+func TestUp_20240813103800(t *testing.T) {
```

Comment:
I would like to run more test cases, but the mechanism to get a clean DB and migrate up to the current migration only (easily) supports a single test per migration. If we have a bit of time near the end of the sprint, I'd try to make that more flexible (though this is an uncommon scenario - it's like a "singleton" in that for a give migration, we can only test one scenario of existing state. usually we can simply create multiple hosts or teams or what have you to test multiple scenarios in one migration test).